mla/test_statistics.py

```diff
@@ -161,9 +161,13 @@ def _preprocess(self, event_model: models.EventModel,
 
 def _sob_time(params: np.ndarray, prepro: _TdPreprocessing) -> float:
     """Docstring"""
-    time_params = prepro.sig_time_profile.param_dtype[:][0]
+    time_params = [name for name, _ in prepro.sig_time_profile.param_dtype]
 
-    sig_time_profile = prepro.sig_time_profile.from_params(params[time_params])
+    if set(time_params).issubset(set(params)):
```

Comment:
if you want to do something like this, write this:
`if set(prepro.sig_time_profile.param_dtype).issubset(set(params.dtype)):`

Reply:
I checked and shouldn't it be if set(time_params).issubset(set(params.dtype))?